src/lib/Containers/__tests__/Conversation-tests.tsx

```diff
@@ -78,14 +78,17 @@ it("sends message when composer is submitted", async () => {
 const props = {
   initials: "JC",
   conversation: {
+    __id: "conversation-420",
     id: "420",
     from: {
       name: "Anita Garibaldi",
       email: "anita@garibaldi.br",
       initials: "AG",
     },
     to: { name: "Kimberly Klark", initials: "KK" },
+    unread: false,
     initial_message: "Adoro! Por favor envie-me mais informações",
+    last_message_id: "222",
     messages: {
```

Comment:
did the relay runtime warn you about these missing fields?

Reply:
Nope, the expanded type definitions did that.